docs/static/ls-to-cloud.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[connecting-to-cloud]]
+=== Sending data to Elasticsearch Service
+
+Our hosted Elasticsearch Service is available on AWS, GCP, and Azure.
+{ess-trial}[You can try the Elasticsearch Service for free].
+
+Logstash comes with two settings that simplify sending data to
+https://cloud.elastic.co/[Elastic Cloud]: Cloud ID and Cloud Auth. 
+
+==== Cloud ID
+
+The Cloud ID, which can be found in the Elastic Cloud web console, is used by
+Logstash to build the Elasticsearch and Kibana hosts settings.
+It is a base64 encoded text value of about 120 characters made up of upper and
+lower case letters and numbers.
+If you have several Cloud IDs, you can add a label, which is ignored
+internally, to help you tell them apart. To add a label you should prefix your
+Cloud ID with a label and a `:` separator in this format "<label>:<cloud-id>"
+
+Cloud ID overwrites these settings:
+----
+var.elasticsearch.hosts
+var.kibana.host
+----
+
+==== Cloud Auth
+This is optional. Construct this value by following this format "<username>:<password>".
+Use your Cloud username for the first part. Use your Cloud password for the second part,
+which is given once in the Cloud UI when you create a cluster.
+As your Cloud password is changeable, if you change it in the Cloud UI remember to change it here too.
+
+Cloud Auth when specified overwrites these settings:
+----
+var.elasticsearch.username
+var.elasticsearch.password
+var.kibana.username
+var.kibana.password
+----
+
+==== Using Cloud ID and Cloud Auth with plugins
+
+The Elasticsearch input, output, and filter plugins support cloud.id and
+cloud.auth in their configurations. 
+
+* <<plugins-inputs-elasticsearch-cloud_id,Elasticsearch input plugin>>
```

Comment:
Should we also include [monitoring](https://www.elastic.co/guide/en/logstash/current/monitoring-internal-collection-legacy.html#monitoring-additional-settings-legacy) and [management](https://www.elastic.co/guide/en/logstash/current/configuring-centralized-pipelines.html#configuration-management-settings)?

Reply:
I added a section for Management. I didn't add legacy collection for monitoring because we're moving away from that.